app/src/main/java/com/itssuryansh/taaveez/NotesEntity.kt

```diff
@@ -11,5 +11,7 @@ data class NotesEntity(
     val Topic: String="",
     val Poem: String="",
     val Date:String="",
-    val CreatedDate:String=""
+    val CreatedDate:String="",
+    val Labels:String=""
```

Comment:
Another table for Labels, and here should be the list of Ids of the labels

Reply:
Bro actually I tried to make List here but during build it was showing some error, that's why I had to store it in string

Reply:
I don't have screenshot of that, by the way you can also check, it will give some error

Reply:
I know about that error, you cannot store Lists in room

Reply:
Having a separate table for Labels and creating a relationship between Notes and Label Table seems a better fit for this use case

Reply:
Bro can you make another issue for that, I don't have experience in databases, I have done other changes you said